<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatRoom</class>
 <widget class="QWidget" name="chatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>191</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Peer List</string>
   </property>
   <widget class="QListWidget" name="peerList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>471</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat area</string>
   </property>
   <widget class="QTextEdit" name="chatArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Type message here</string>
   </property>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chatMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="roomDescTopic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
